src/service/fileMaker.ts

```diff
@@ -60,3 +86,25 @@ export async function makeMigration(config: Configuration, filename: string): Pr
 
   return [upFilename, downFilename];
 }
+
+/**
+ * Generate JS/TS migration file(s).
+ *
+ * @param {string} filename
+ * @param {string} migrationPath
+ * @param {string} timestamp
+ * @param {string} extension
+ * @returns {Promise<string[]>}
+ */
+export async function makeJSMigration(
+  filename: string,
+  migrationPath: string,
+  timestamp: string,
+  extension: string = FileExtensionTypes.JS
+): Promise<string[]> {
+  const migrationFilename = path.join(migrationPath, `${timestamp}_${filename}.${extension}`);
+
+  await fs.write(migrationFilename, '');
```

Comment:
Don't make it empty. Use a stub/template file like above - pretty straightforward. Or, at least add a TODO comment if it's still a WIP.

Reply:
Stubs found here:

* For TS: https://github.com/laudio/db/blob/dev/src/stubs/migration.stub
* For JS: https://github.com/laudio/db/blob/dev/system/src/stubs/migration.stub